apps/vanilla/src/interface/paginationInterface.ts

```diff
@@ -0,0 +1,33 @@
+import { SortTitle, SortValue } from '@js-camp/core/enum';
+
+/** Sorting info. */
+export interface Sorting {
+
+  /** Title of sorting. */
+  title: SortTitle;
```

Comment:
```suggestion
  readonly title: SortTitle;
```
[Promote immutability](https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices/#promote-immutability)
As we already suggested in [code practices section](https://wiki.saritasa.rocks/frontend/practices/code#promote-immutability), you should always make sure you're operating with immutable data objects. Typescript allows to explicitly declare more data this way, that is exactly what we need 🐱‍🏍.

Use readonly as much as you use const in order to explicitly declare the variables/properties immutable.

If you want to change your value, please use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#examples). Feel free to ping me anytime in case you have any questions or suggestions